packs/data/extinction-curse-bestiary.db/shoony-hierarch.json

```diff
@@ -236,9 +495,239 @@
             },
             "img": "systems/pf2e/icons/default-icons/equipment.svg",
             "name": "staff",
```

Comment:
Delete

Reply:
Fixed in [5f3c718](https://github.com/foundryvtt/pf2e/pull/2810/commits/5f3c718c6b78d33f4c335a7b9398895043044537)